### PR TITLE
Todo Listing Page

### PR DESCRIPTION
**Pull request by ArjunGTX**

### CODE REVIEW

File: src/components/TodoCard.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      toast.error(toastError.GENERAL);
+    }
+  };
+
+  const deleteTodoRequest = async () => {
+    try {
+      const { status, data } = await api.deleteTodo(todo._id);
+      if (status !== 200) return;
+      syncTodosWithServer();
+      console.log(data);
```
Comment (ashishnnnnn) [inline src/components/TodoCard.jsx]: Console can be removed.. if not required

File: src/components/TodoModal.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    todo ? updateTodoRequest() : createTodoRequest();
+  };
+
+  const handleInputChange = (e) =>
+    setTodoInputs((inputs) => ({
+      ...inputs,
+      [e.target.id]: e.target.value,
+    }));
```
Comment (ashishnnnnn) [inline src/components/TodoModal.jsx:65]: Liked this way to handle the form input..

**Review by ashishnnnnn — APPROVED** [review]
Code looks good..
Good Work..

**Review by nerdyspook — APPROVED** [review]
Code is clean and well structured.

File: src/components/TodoCard.jsx (hunk 1 above)
Comment (ArjunGTX) [reply src/components/TodoCard.jsx]: done, thankyou for pointing out

**Pull request merged**